gui/src/components/TableButtons.js

```diff
@@ -0,0 +1,46 @@
+import { IconButton, withStyles } from '@material-ui/core'
+import FirstPageIcon from '@material-ui/icons/FirstPage'
+import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft'
+import KeyboardArrowRight from '@material-ui/icons/KeyboardArrowRight'
+import LastPageIcon from '@material-ui/icons/LastPage'
+import React from 'react'
+
+const styles = theme => ({
+  customButtons: {
+    flexShrink: 0,
+    color: theme.palette.text.secondary,
+    marginLeft: theme.spacing.unit * 2.5
+  }
+})
+
+const TableButtons = props => {
+  const firstPage = 1
+  const currentPage = props.page
+  const lastPage = Math.ceil(props.count / props.rowsPerPage)
+  const handlePage = page => e => {
+    page = Math.min(page, lastPage)
+    page = Math.max(page, firstPage)
+    if (props.history) props.history.replace(`${props.replaceWith}/${page}`)
```

Comment:
@rupurt What do you think about this instead of wrapping IconButtons in <Link/>  ?

Reply:
@NavyAdmiral I'm not quite sure what you mean by `instead of wrapping IconButtons in ?`

But I like this component for `TableButtons` in general